README.md

```diff
@@ -93,26 +93,41 @@ Set your Ruby version with `rbenv`.
 ```bash
 brew install rbenv
 rbenv versions # see which ruby versions are already installed
-rbenv install 2.6 # install 2.6 if you don't have it already
-rbenv local 2.6 # rbenv local --unset reverses the action
+rbenv install  # the app currently uses version 2.6.10
+rbenv local # rbenv local --unset reverses the action
+
+# To switch between rbenv versions installed locally, use the following command:
+rbenv shell 2.6.10
+
 ```
 
 Set your Node version with `NVM`.
 
 ```bash
 brew install nvm
+brew info nvm # commands that shows the remaining steps to complete to install nvm properly
+mkdir ~/.nvm
 nvm install 14
 nvm use 14
+# Add the following lines to your ~/.bashprofile or ~/.zshrc:
 echo 'export NVM_DIR="$HOME/.nvm"' >> ~/.zshrc
 echo '[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm' >> ~/.zshrc
 echo '[ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"  # This loads nvm bash_completion' >> ~/.zshrc
+
+# Reference Stack Overflow post: https://stackoverflow.com/questions/53118850/brew-install-nvm-nvm-command-not-found
 ```
 
 Set your Postgres version with homebrew.
 
 ```bash
 brew install postgresql@12
 brew switch postgres@12
+
+# To start postgres locally run:
+brew services start postgresql@12
+
+# To stop postgres locally run:
+brew services stop postgresql@12
```

Comment:
I don't think I understand why this line is here. If they do this, they won't be able to use the software, right?

Reply:
Ah. Well we don't want it here because then users can't copy and paste the entire script as-is.